src/ChangeStream.php

```diff
@@ -201,6 +191,30 @@ private function extractResumeToken($document)
         return $resumeToken;
     }
 
+    /**
+     * Determines if an exception is a resumable error.
+     *
+     * @see https://github.com/mongodb/specifications/blob/master/source/change-streams/change-streams.rst#resumable-error
+     * @param RuntimeException $exception
+     * @return boolean
+     */
+    private function isResumableError(RuntimeException $exception)
```

Comment:
The criteria for a resumable error changed since the spec was first implemented in 1.3.0. See: [Resumable Error](https://github.com/mongodb/specifications/blob/master/source/change-streams/change-streams.rst#resumable-error).